src/routes.ts

```diff
@@ -0,0 +1,24 @@
+import { HomeController } from './controllers/home';
+import { BaseController } from './controllers/base';
+import { Context } from 'koa';
+
+const Router = require('koa-router');
+const router = new Router();
+
+type RouteDefinition = [string, BaseController];
+
+let routes: Array<RouteDefinition>;
+routes = [
```

Comment:
why not an object?

Reply:
You mean a key->value like thing? It could make sense, but typing seems a bit better if you use 'array of type', especially if you end up looping over it.

Reply:
You can, but I find the syntax a bit more obtuse. An array of tuples feels a bit more natural for this kind of thing.

Reply:
Pretty subjective, yea

Reply:
alrighty, yeah I'm more just curious